tools/perf/ResultsComparer/Program.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
We haven't migrated odata.net to .NET foundation

Reply:
Well the project is a member so I think this not wrong.